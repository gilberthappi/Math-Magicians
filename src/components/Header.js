import { Link } from 'react-router-dom';

const Header = () => (
  <header>
    <div className="logo">
      <Link to="/">Math Magicians</Link>
    </div>
    <nav className="navbar">
      <ul className="navLinks">
        <li>
          <Link to="/">Home</Link>
        </li>
        <li>|</li>
        <li>
          <Link to="/Calculator">Calculator</Link>
        </li>
        <li>|</li>
        <li>
          <Link to="/Quote">Quote</Link>
        </li>
      </ul>
    </nav>
  </header>
);

export default Header;
