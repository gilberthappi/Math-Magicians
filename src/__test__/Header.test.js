import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Header from '../components/Header';

describe('Header component testing', () => {
  it('Renders correctly', () => {
    const tree = renderer
      .create(<BrowserRouter><Header /></BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
