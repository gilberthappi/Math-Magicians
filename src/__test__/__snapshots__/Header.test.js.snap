// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component testing Renders correctly 1`] = `
<header>
  <div
    className="logo"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Math Magicians
    </a>
  </div>
  <nav
    className="navbar"
  >
    <ul
      className="navLinks"
    >
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        |
      </li>
      <li>
        <a
          href="/Calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        |
      </li>
      <li>
        <a
          href="/Quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
